test/components/dragBoxLayerTests.ts

```diff
@@ -61,45 +82,46 @@ describe("Layer Components", () => {
         assert.strictEqual(dbl.detectionRadius(5), dbl, "setting the detection radius returns the drag box layer");
         assert.strictEqual(dbl.detectionRadius(), 5, "can retrieve the detection radius");
 
-        assert.throws(() => dbl.detectionRadius(-1), "detection radius cannot be negative");
+        (<any> assert).throws(() => dbl.detectionRadius(-1), Error, "detection radius cannot be negative", "fails to set negative radius");
```

Comment:
Can you tag this with the appropriate HACKHACK (https://github.com/palantir/plottable/issues/2661) ?